content/en/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Configuring a cgroup driver
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+This page explains how to configure the kubelet cgroup driver to match the container
+runtime cgroup driver for kubeadm clusters.
+
+## {{% heading "prerequisites" %}}
+
+You should be familiar with the Kubernetes
+[container runtime requirements](/docs/setup/production-environment/container-runtimes).
+
+<!-- steps -->
+
+## Configuring the container runtime cgroup driver
+
+The [Container runtimes](/docs/setup/production-environment/container-runtimes) page
+provides details on how to setup a number of different container runtimes with the
+`systemd` driver by default.
+
+The same page explains that the `systemd` driver is recommended for kubeadm based setups
```

Comment:
What about moving this sentence and the beginning of the paragraph so first we are telling the user they should use systemd and why, and then we explain how.

Reply:
makes sense.